authors/apps/authentication/urls.py

```diff
@@ -1,11 +1,15 @@
 from django.urls import path
-
+from rest_framework_swagger.views import get_swagger_view
+from rest_framework.documentation import include_docs_urls
```

Comment:
@akram256 Move these two swagger and documentation imports to the urls module of the project.

Reply:
I have worked on the feed back